piqe_ocp_lib/api/resources/ocp_operators.py

```diff
@@ -200,6 +200,19 @@ def get_package_ownnamespace_channel(self, package_name):
             logger.error("A OwnNamespace channel was not found for package: {}".format(package_name))
         return None
 
+    def get_yaml_from_annotation(self, channel_name):
```

Comment:
I suggest renaming this method to something more descriptive. Unless the end user is familiar with the schema, this name will not be very clear. I think something like "get_crd_models_from_manifest()" might be a little more clear. The distinction is describing what we are getting, and not where we are getting it from.
Also note that in addition to 'channel_name', this method needs to take 'operator_name' as an input parameter as well.

Reply:
done